end-to-end-test/tests/onboarding/onboarding.ts

```diff
@@ -0,0 +1,164 @@
+import { Page, expect, chromium } from '@playwright/test';
+
+const suspend = (value: number) => new Promise(resolve => setTimeout(resolve, value));
+
+export default class OnboardingFlow {
+    readonly page: Page;
+    readonly email: string;
+    private signupPage: Page | null;
+
+    constructor(page: Page) {
+        const randomString = new Date().getTime();
+        this.page = page;
+        this.email = `deriv-fe-e2e-${randomString}@deriv.com`;
+        this.signupPage = null;
+    }
+
+    async changeEndpoint() {
```

Comment:
IMO, `changeEndpoint` shouldn't be part of Onboarding. It should be separated so that it can be used in other places too.